lib/index.ts

```diff
@@ -127,7 +129,7 @@ export function hasConflict(files: (File|FileSystemEntry|Node)[], content: Node[
 export function getConflicts<T extends File|FileSystemEntry|Node>(files: T[], content: Node[]): T[] {
 	const contentNames = content.map((node: Node) => node.basename)
 	const conflicts = files.filter((node: File|FileSystemEntry|Node) => {
-		const name = (node instanceof File || node instanceof FileSystemEntry) ? node.name : node.basename
+		const name = (node instanceof File || isFileSystemEntry(node)) ? node.name : node.basename
```

Comment:
Indeed it is not defined... But `isFileSystemEntry` not checks if it is available, and returns false eve node is `FileSystemEntry`. Why is it fine? Node is instance of `File` on Chromium?

Reply:
Just to prevent the exception every other behavior is the same. But see comment below

Reply:
I think the otherway round would be safer, checking if node is instance of `Node` (so either `Node`, `File` or `Folder` from Nextcloud files).

Changed the PR.